workspaces/repo-tools/packages/cli/src/lib/workspaces/templates/workspace/package.json.hbs

```diff
@@ -35,9 +35,8 @@
   "devDependencies": {
     "@backstage/cli": "^{{version '@backstage/cli'}}",
     "@backstage/e2e-test-utils": "^{{version '@backstage/e2e-test-utils'}}",
-    "@backstage/repo-tools": "^0.10.0",
+    "@backstage/repo-tools": "^{{version '@backstage/repo-tools'}}",
     "@changesets/cli": "^2.27.1",
-    "@spotify/prettier-config": "^12.0.0",
     "knip": "^5.27.4",
     "node-gyp": "^9.0.0",
```

Comment:
Can we update this to 10 while we are at it: https://github.com/backstage/backstage/blob/b38b32fbbff95e9b7319de3c479a0f4ba2253c9c/packages/create-app/templates/default-app/package.json.hbs#L37